modules/porous_flow/doc/content/modules/porous_flow/sqa/porous_flow_sdd.md

```diff
@@ -1 +1,65 @@
 !template load file=sqa/module_sdd.md.template category=porous_flow module=Porous Flow
+
+!template! item key=introduction
+The [!ac](MOOSE) {{module}} module is based on the MOOSE framework and thus inherits
+the unique features and base characteristics of the framework, as outlined in the [framework_sdd.md].
+Specific details unique to the module are outlined in this document.
+!template-end!
+
+!template! item key=system-scope
+!include porous_flow_srs.md start=system-scope-begin end=system-scope-finish
+!template-end!
+
+!template! item key=dependencies-and-limitations
+The {{module}} module inherits the
+[software dependencies and limitations of the MOOSE framework](framework_sdd.md#dependencies-and-limitations).
+The {{module}} module currently has the following limitations in functionality...
+!template-end!
+
+!template! item key=design-stakeholders
+!include framework_sdd.md start=design-stakeholders-begin end=design-stakeholders-finish
+!template-end!
+
+!template! item key=system-design
+The {{module}} module inherits the wide range of pluggable systems from MOOSE. More
+information regarding MOOSE system design can be found in the framework [framework_sdd.md#system-design]
+section.
+
+The {{module}} module provides functionality to support simulations for fluid and heat flow in porous
+media. More detailed information on the theory of the module can be found on the [module home page](porous_flow/index.md)
+under the subsection "Module overview". Numerical implementation details for the module can be found
+on the same page, under the subsection "Implementation details". A requirement of each {{module}}
+simulation is the [`PorousFlowDictator`](porous_flow/dictator.md) object, which holds information about
+the nonlinear solution variables within the module, as well as the number of fluid phases and fluid
+components in each simulation.
```

Comment:
I took a whack at describing the module structurally, but I know there's a lot more to cover here. This would be the spot to briefly describe notable sets of kernels, boundary/interface conditions, objects, etc. that give the module a significant portion of its functionality. As you can see, pointing the reader out to an external document is OK, as long as it fits this theme.